_1327/main/migrations/0005_add_anonymous_group.py

```diff
@@ -2,15 +2,26 @@
 from __future__ import unicode_literals
 
 from django.db import migrations
-from django.contrib.auth.models import Group
 from guardian.management import create_anonymous_user
 from guardian.utils import get_anonymous_user
+from guardian.conf import settings as guardian_settings
+
+from _1327.user_management.models import UserProfile
+
 
 def add_anonymous_group(apps, schema_editor):
-	create_anonymous_user(None)
+	User = apps.get_model("user_management", "UserProfile")
+	user_arguments = {UserProfile.USERNAME_FIELD: guardian_settings.ANONYMOUS_USER_NAME}
+	anonymous_user = User(**user_arguments)
+	# anonymous_user.set_unusable_password()
```

Comment:
please set this line to `anonymous_user.password = make_password(None)`
```suggestion
	anonymous_user.password = make_password(None)
```
and also
```suggestion
        from django.contrib.auth.hashers import make_password
```